packages/compat/src/v1-app.ts

```diff
@@ -135,6 +135,16 @@ export default class V1App implements V1Package {
     return this.app.options.storeConfigInMeta;
```

Comment:
Could we ensure the new code-path is tested?

Reply:
I added the tests as part of macro-sample-addon to test the code path for config-module.
@stefanpenner @ef4